src/components/InputTodo.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/prop-types */
+import { useState } from 'react';
+
+const InputTodo = ({ addTodoItem }) => {
+  const [title, setTitle] = useState('');
+  const [message, setMessage] = useState('');
+
+  const handleChange = (e) => {
+    setTitle(e.target.value);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (title.trim()) {
+      addTodoItem(title);
+      setTitle('');
+      setMessage('');
+    } else {
+      setMessage('Please add item.');
+    }
+  };
+  return (
+    <>
+      <form onSubmit={handleSubmit} className="form-container">
+        <input
+          type="text"
+          placeholder="Add Todo..."
+          value={title}
+          onChange={handleChange}
+          className="input-text"
+        />
+        <button type="button" className="input-submit">Submit</button>
```

Comment:
Changing the button type to "submit", will insure that the button will work in addition to the Enter press

Reply:
That's a great idea. Let me apply it immediately